Tasks/ServiceFabricDeploy/deploy.ps1

```diff
@@ -69,5 +90,10 @@ try {
         Publish-NewServiceFabricApplication -ApplicationPackagePath $ApplicationPackagePath -ApplicationParameterFilePath $publishProfile.ApplicationParameterFile -Action RegisterAndCreate -OverwriteBehavior SameAppTypeAndVersion -ErrorAction Stop 
     }
 } finally {
+    if ($certificate)
+    {
+        Remove-Certificate -ClusterConnectionParameters $clusterConnectionParameters -Certificate $certificate
```

Comment:
Yep.  The correct way to do this is to just leave the certificate installed, same as is done for Azure certificates on Azure-related tasks.